frontend/js/player-adapter-html5.js

```diff
@@ -173,6 +173,11 @@ define(
                             e.preventDefault();
                         });
 
+                        // Workaround to fix chrome fullscreen bug (#214)
+                        $(document).bind('fullscreenchange webkitfullscreenchange mozfullscreenchange msfullscreenchange', function (e) {
+                            self.resetSize();
```

Comment:
The video does not scale/resize correctly in browsers; see also https://github.com/opencast/annotation-tool/issues/206#issuecomment-478961780

This reduces the issue